modules/layers/src/arc-layer/arc-layer-vertex.glsl.js

```diff
@@ -198,8 +198,14 @@ void main(void) {
     source = project_position(source_world, instanceSourcePositions64Low);
     target = project_position(target_world, instanceTargetPositions64Low);
 
+    // common x at longitude=-180
+    float antiMeridianX = 0.0;
+
     if (useShortestPath) {
-      float thresholdRatio = -source.x / (target.x - source.x);
+      if (project_uProjectionMode == PROJECTION_MODE_WEB_MERCATOR_AUTO_OFFSET) {
```

Comment:
We have the following condition in project.glsl. Not exactly sure, but do we need the same check for antiMeridianX?

```
if (project_uProjectionMode == PROJECTION_MODE_IDENTITY ||
    (project_uProjectionMode == PROJECTION_MODE_WEB_MERCATOR_AUTO_OFFSET &&
    (project_uCoordinateSystem == COORDINATE_SYSTEM_LNGLAT ||
     project_uCoordinateSystem == COORDINATE_SYSTEM_CARTESIAN))) {
    // Subtract high part of 64 bit value. Convert remainder to float32, preserving precision.
    position_world.xyz -= project_uCoordinateOrigin;
  }
```

Reply:
`wrapLongitude` is documented as only works with the LNGLAT coordinate system: https://deck.gl/docs/api-reference/core/layer#wraplongitude